extensions/lekab/src/module.ts

```diff
@@ -0,0 +1,14 @@
+import { createExtension } from "@cognigy/extension-tools";
+
+import { sendSMSNode } from "./nodes/sendSMSNode";
+import { lekabConnection } from "./connections/lekabConnection";
+
+export default createExtension({

```

Comment:
Please provide the "options" property in order to give it a display name such as shown in this example: https://github.com/Cognigy/Extensions/blob/master/extensions/8x8/src/module.ts#L18

Reply:
options added.